jsonnetfile.lock.json

```diff
@@ -34,27 +34,27 @@
     {
       "source": {
         "git": {
-          "remote": "https://github.com/grafana/grafonnet-lib.git",
+          "remote": "https://github.com/grafana/grafonnet-lib",
           "subdir": "grafonnet"
         }
       },
-      "version": "5c6e8a8113486cdecd0961730aeaada3e6c69fe7",
-      "sum": "tDuuSKE9f4Ew2bjBM33Rs6behLEAzkmKkShSt+jpAak="
+      "version": "3336c69715f8f7a4d637582504c9fabd9d9ca081",
+      "sum": "w6zS28Rjs9EzRN/WoLLIdi028BvumxDTyLefYVoql2k="
     },
     {
       "source": {
         "git": {
-          "remote": "https://github.com/grafana/jsonnet-libs.git",
+          "remote": "https://github.com/grafana/jsonnet-libs",
```

Comment:
Did you use jb `v0.3.1`?

Reply:
I used `make generate` which uses a local `jb` in `tmp/bin/jb`, pinned by `go.mod.`

Reply:
```
$ make generate
Installing tools from scripts/tools.go
go install github.com/brancz/gojsontoyaml
go install github.com/campoy/embedmd
go install github.com/google/go-jsonnet/cmd/jsonnet
go install github.com/google/go-jsonnet/cmd/jsonnetfmt
go install github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb
rm -rf vendor
/home/sur/src/redhat/src/kube-prometheus/tmp/bin/jb install
....
```